src/container/AppBoot/index.tsx

```diff
@@ -13,14 +13,17 @@ import 'react-toastify/dist/ReactToastify.css';
 function App() {
   return (
     <Provider store={ReduxStore}>
+      {/* PersistGate delays the rendering of the app UI till the persisted state has been retrieved and saved to redux */}
       <PersistGate loading={null} persistor={persistor}>
         <div className="app-container">
+          {/* Used to preserve previous component path */}
           <ConnectedRouter history={history}>
             <AppRoutes />
           </ConnectedRouter>
         </div>
 
-        <ToastContainer
+        {/* react-toastify is used to add notifications */}
+        {/* <ToastContainer
```

Comment:
@alokesh985 Why is this ToastContainer commented?

Reply:
I had commented it out for testing something. Uncommented now.